src/cephadm/tests/test_cephadm.py

```diff
@@ -1575,15 +1575,15 @@ def test_name(self, cephadm_fs):
             retval = _cephadm.command_shell(ctx)
             assert retval == 0
 
-        cmd = ['shell', '--name', 'foo.bar']
+        cmd = ['shell', '--name', 'mgr.bar']
```

Comment:
Out of curiosity, what made this become necessary now?

Reply:
The code now relies on mapping the deamon type to a class via the daemon_form.create function. Since foo isn't an actual daemon type name that maps to a class the function fails. This changes it to use a valid daemon_type.